scraper/load.py

```diff
@@ -0,0 +1,526 @@
+load_dotenv()
+
+
+class DataLoader:
+    def __init__(self, api_url, api_key, results_number) -> None:
+        self.api_url = api_url
+        self.api_key = api_key
+        self.category_map = {}
+        self.atribbs_map = {}
+        self.features_map = {}
+        self.results_number = results_number
+        self.base_weight_price = 999
+        self.find_base_weight()
+        self.load_category("Włóczki", 2)
+
+    def start(self):
+        self.find_base_weight()
+        for i in range(1, self.results_number+1):
+            with open(f'results/products_{i}.json', 'r', encoding='utf-8') as file:
+                data = json.load(file)
+                self.handle_json(data)
+
+    def handle_json(self, data):
+        for product in data:
+            categories_tree = product.get('categories_tree', [])
+            self.load_categories(categories_tree)
+            self.load_product(product)
+
+    def find_base_weight(self):
+        for i in range(1, self.results_number+1):
+            with open(f'results/products_{i}.json', 'r', encoding='utf-8') as file:
+                data = json.load(file)
+                for product in data:
+                    prices = product["prices"]
+                    weights = prices["weights"]
+                    if weights:
+                        min_weight_price = min(prices["netto"])
+                        self.base_weight_price = min(min_weight_price, self.base_weight_price)
+              
+    def load_category(self, cat_name, parent_id):
+        if cat_name in self.category_map:
+            print("Kategoria już jest obecna")
+            print("\n", end="")
+            return
+
+        request_xml = f"""<?xml version="1.0" encoding="UTF-8"?>
+            <prestashop xmlns:xlink="http://www.w3.org/1999/xlink">
+                <category>
+                    <name>
+                        <language id="1"><![CDATA[{cat_name}]]></language>
+                    </name>
+                    <link_rewrite>
+                        <language id="1"><![CDATA[{cat_name}]]></language>
+                    </link_rewrite>
+                    <description>
+                        <language id="1"><![CDATA[{cat_name} opis]]></language>
+                    </description>
+                    <active>1</active>
+                    <id_parent>{parent_id}</id_parent>
+                </category>
+            </prestashop>"""
+
+        response = self.make_request("POST", "categories", request_xml)
+
+        if response.status_code == 201:
+
+            response_xml = ET.fromstring(response.text)
+            cat_id = response_xml.find('.//id').text
+
+            print("Kategoria została pomyślnie dodana.")
+            print(f"ID kategorii: {cat_id}")
+            print(f"Nazwa kategorii: {cat_name}")
+
+            self.category_map[cat_name] = cat_id
+        else:
+            print(f"Błąd dodawania kategorii: {response.status_code} - {response.text}")
+
+        print("\n", end="")
```

Comment:
This also can be simplified
```suggestion
        print()
```